tests/components/unifi_direct/test_device_tracker.py

```diff
@@ -74,7 +74,7 @@ async def test_get_device_name(mock_ssh, hass):
             CONF_CONSIDER_HOME: timedelta(seconds=180),
         }
     }
-    mock_ssh.return_value.before = load_fixture("unifi_direct.txt")
+    mock_ssh.return_value.before = load_fixture("direct.txt", "unifi")
```

Comment:
The unifi_direct integration is separate from the unifi integration. The fixture was moved to the wrong place.